src/app.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express');
+const { join } = require('path');
+const cookiePerser = require('cookie-parser')
+const rauter = require('./router');
+const { e404, serverErr } = require('./controllers/err');
```

Comment:
relate to this issue please [#29](https://github.com/GSG-G9/IM-weather-app/issues/29)

Reply:
is this better?